cargo-shuttle/src/lib.rs

```diff
@@ -405,7 +406,8 @@ impl Shuttle {
             secrets,
             working_directory.to_path_buf(),
         )?;
-        let addr = SocketAddr::new(Ipv4Addr::LOCALHOST.into(), run_args.port);
+        let addr = SocketAddr::new(if run_args.router_ip 
```

Comment:
This is cool, but probably not quite the Rust way to use the `if`. It also reads a bit hard. Maybe do something along the lines of:

```rust
let addr = if run_args.router_ip {
         local_ip().unwrap()
      } else {
         Ipv4Addr::LOCALHOST.into()
      };
let addr = SocketAddr::new(addr, , run_args.port);
```

Reply:
Ah thanks for catching that, I have fixed as per requested - once the amendments are ready I'll push them through